pkg/build/node/cni.go

```diff
@@ -185,21 +186,23 @@ spec:
       serviceAccountName: kindnet
       containers:
       - name: kindnet-cni
-        image: kindest/kindnetd:0.2.0
+        image: kindest/kindnetd:0.3.0
         env:
-          - name: HOST_IP
-            valueFrom:
-              fieldRef:
-                fieldPath: status.hostIP
-          - name: POD_IP
-            valueFrom:
-              fieldRef:
-                fieldPath: status.podIP
-          - name: CNI_CONFIG_TEMPLATE
-            valueFrom:
-              configMapKeyRef:
-                name: kindnet-cfg
-                key: cni-conf-template.json
+        - name: HOST_IP
```

Comment:
is this well indented? seem indentation has changed

Reply:
this normalized the indentation, see EG the volumeMounts below